atom/browser/api/atom_api_menu.cc

```diff
@@ -40,6 +40,7 @@ void Menu::AfterInit(v8::Isolate* isolate) {
   delegate.Get("isCommandIdChecked", &is_checked_);
   delegate.Get("isCommandIdEnabled", &is_enabled_);
   delegate.Get("isCommandIdVisible", &is_visible_);
+  delegate.Get("isCommandIdWorksWhenHidden", &is_works_when_hidden_);
```

Comment:
/nit This is like english semantics but can this be `doesCommandIdWorkWhenHidden` so that it reads as a question like the other methods above

Reply:
Same throughout the PR if possible, not too set on it just an IMO thing

Reply:
done